CHANGELOG.md

```diff
@@ -84,6 +84,5 @@ release.
   ([#55](https://github.com/open-telemetry/semantic-conventions/pull/55))
 - Split out sections for proposed stable JVM metrics and experimental JVM metrics.
   ([#56](https://github.com/open-telemetry/semantic-conventions/pull/56))
-- BREAKING: Remove `messaging.source.*` attributes and use `messaging.destination.*`
-  attributes on producer and consumer to describe messaging queue or topic.
+- BREAKING: Remove `messaging.source.*` attributes.
```

Comment:
I think the previous changelog was better?

Reply:
Well, before we simply renamed them, so "what it was for" still applied. But now that it is gone, not sure what else we can indicate. Maybe

> Remove `messaging.source` attributes to not clash with ECS `source` namespace? Because that's the main driver for this change IIRC.

Reply:
Ah I see now, you are right. I got to excited in removing things it seems. Re-added 👍